xtext-website/documentation/103_domainmodelnextsteps.md

```diff
@@ -235,46 +285,59 @@ This tutorial is about testing the parser and the linker for the *Domainmodel*.
             assertSame(entity, entity.features.head.type)
         }
     ```
+2. In addition, the utility class [ValidationTestHelper]({{site.src.xtext_core}}/org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/validation/ValidationTestHelper.java) allows to test the custom validation rules written for the language. Both valid and invalid models can be tested.
+
+    ```xtend      
+	@Inject ValidationTestHelper validationTestHelper
+
+	@Test
+	def testNameStartsWithCapitalWarning() {
+        val entity = parseHelper.parse(
+            "entity myEntity {
+                parent: myEntity
+            }")
+        validationTestHelper.assertWarning(entity, DomainmodelPackage.Literals.ENTITY, 
+            DomainmodelValidator.INVALID_NAME)
+	}
```

Comment:
I think it would be a good idea to verify the warning message in the test case:
```
validationTestHelper.assertWarning(entity, DomainmodelPackage.Literals.ENTITY,
    DomainmodelValidator.INVALID_NAME, "Name should start with a capital")
```